src/jquery-extensions.js

```diff
@@ -0,0 +1,12 @@
+(function ($) {
+    var originalVal = $.fn.val;
+    $.fn.val = function(){
+        if (this.hasClass('fake-textarea') || this.hasClass('fake-input')) {
+            return $.fn.text.apply(this, arguments);
+        }
+        var result = originalVal.apply(this,arguments);
+        if(arguments.length>0)
+            $(this).change();
```

Comment:
You caught this right as I did. It's already updated and won't be there. (copied from SO)